README.md

```diff
@@ -1,6 +1,27 @@
 # lambda-sc-cost-meter
 Lambda to report Service Catalog costs to the AWS Marketplace
 
+## Description
+
+This app will look for resources in an AWS account that contains tags indicating that
+the resources were provisioned by a Service Catalog subscriber.  SC subscriber resources
```

Comment:
the resources were provisioned by a AWS Marketplace subscriber.  Marketplace subscriber resources

Reply:
fixed